base/.env

```diff
@@ -22,6 +22,6 @@ NDSSCPIMEGA_VERSION=1.0.0
 LIBSSCPIMEGA_VERSION=fb8acf533a7c01b5266bf32d60d1a5f923e19523
 
 MOTOR_VERSION=R7-3-1
-PMAC_VERSION=2-6-1
+PMAC_VERSION=2-6-4b3
```

Comment:
As far as I can tell, these look like pre-releases. Do we need any of the included changes?

Reply:
There was no need for changes, just updating the version with corrections to the module.

Reply:
I meant, do we need any of the changes between 2-6-1 and 2-6-4b3? Could we use 2-6-3, or wait for the release of 2-6-4 proper?

Reply:
> Do we need any of the included changes?

Yes, we need https://github.com/dls-controls/pmac/commit/7ab9f2413d60ce77c1ba4c32c256253e0535ef7e, which prevents a segfault. @guirodrigueslima told me the problem possibly happens when disconnection a occurs (even though we haven't checked this in details).

> wait for the release of 2-6-4 proper?

@guirodrigueslima can answer this better than me, but I understand he wants to deploy this ASAP.

Reply:
There are two problems with the current version (2-6-1), the first is a "segmentation fault", when the coordinate system axes object does not exist, I don't know how to reproduce this problem, it was observed by Diamond Light Source. The other problem is related to the loss of connection with delta tau, when connecting again the offsets receive null values.

Reply:
In that case, I think specifically the Pmac update would fit best under "Bug fixes".

And mention in the commit message that we are using a pre-release because we need the specific fixes and there hasn't been a release yet.